TunnelKit/Sources/Protocols/OpenVPN/ConfigurationParser.swift

```diff
@@ -517,6 +518,17 @@ extension OpenVPN {
                     optSearchDomain = $0[1]
                 }
                 Regex.proxy.enumerateArguments(in: line) {
+                    if $0.count == 2 {
+                        let maybeURL = URL(string: $0[1])
+                        if maybeURL != nil {
```

Comment:
Any reason against an `if let`? Or `guard let`, much better.

Reply:
Yeah XCode was buggy and when I debugged the test it was showing the url as `nil` so I wrote this (which didn't solve the XCode issue) but forgot to change it back!